NuWeather/plugin.py

```diff
@@ -147,13 +147,13 @@ def _format_temp(self, f, c=None):
 
         displaymode = self.registryValue('units.temperature', dynamic.msg.args[0])
         if displaymode == 'F/C':
-            string = '%sF/%sC' % (f, c)
+            string = '%s°F/%s°C' % (f, c)
         elif displaymode == 'C/F':
-            string = '%sC/%sF' % (c, f)
+            string = '%s°C/%s°F' % (c, f)
         elif displaymode == 'F':
-            string = '%sF' % f
+            string = '%s°F' % f
         elif displaymode == 'C':
-            string = '%sC' % c
+            string = '%s°C' % c
```

Comment:
I don't have a strong opinion but adding the degree symbol to every temperature string takes up a decent amount of space. With IRC message length limits, every byte counts...

I would rather replace this with a templating system that allows for more flexibility, or make this option accept strings like `'°F/C'`, `'°C'`, etc.

Reply:
Agreed! That space would be less of a premium if we stripped the URL, and just attributed to OpenWeatherMap. OSM/Nominatim should not be there if not used, like in my case with Google + OWM.

Reply:
Yeah I'd be open to removing the URL from the default format. The geocoding backends do have to be credited separately though because they're different services - e.g. see [Nominatim's usage policy](https://operations.osmfoundation.org/policies/nominatim/)

Reply:
The separate backend attribution is not something I wish to muck with, should I just delete this PR and let you add those things in, my suggestions are rather minor in comparison?

Reply:
Yeah, fair enough